doc/api/http.md

```diff
@@ -1443,6 +1443,31 @@ This event is guaranteed to be passed an instance of the {net.Socket} class,
 a subclass of {stream.Duplex}, unless the user specifies a socket
 type other than {net.Socket}.
 
+### `server.activeConnections()`
```

Comment:
A request is not a connection. A request is a communication across a connection. I think `server.activeConnections` is misleading. According to the subsequent docs, it is `server.activeRequests`.

Reply:
Nope, it's connections. I return the sockets, even though the way the sockets are categorized is given by whether there is an active/expired request being sent through it.

Reply:
How does this help with draining connections? The `socket` does not expose the requests associated with it.

Reply:
I see your point now.
Let me clarify that, of all these methods, only `allConnections` and `idleConnections` are strictly needed to properly implement the forced close logic.

But I do see your point and, on a second thought, it makes sense to expose the requests rather than socket. This way the developer has all the needed informations to implement any logic they might want.

Reply:
#2 is hard as it needs a check every time a request is finished. Which will impact performance. So eventually (which is the intention of this PR) we go with 1 (as suggested in my last comment).